drivers/w5500/include/w5500_params.h

```diff
@@ -41,8 +41,8 @@ extern "C" {
 #ifndef W5500_PARAM_INT
 #define W5500_PARAM_INT         GPIO_UNDEF          /**< set to invalid */
 #endif
-#ifndef CONFIG_W5500_POLLING_INTERVAL
-#define CONFIG_W5500_POLLING_INTERVAL 100u          /**< default polling interval 100 ms */
+#ifndef W5500_POLLING_INTERVAL_MS
+#define W5500_POLLING_INTERVAL_MS 100u              /**< default polling interval 100 ms */
```

Comment:
No need to drop the `CONFIG_` prefix, it's still used to indicate defines that can be overwritten by the user.

Reply:
Why do the other parameters not have the prefix?

Reply:
One answer is consistency, but a better answer is that they are overwritten by the board, not the user.